README.md

```diff
@@ -68,44 +68,40 @@ GRANT ALL PRIVILEGES on syncstorage_rs.* to sample_user@localhost;
 
 ### Spanner
 
-Spanner requires a key in order to access the database. It's important that you know which keys have access to the spanner database. Contact your administrator
-to find out. One you know the key, log into the [Google Cloud Console Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts) page. Be sure to
-select the correct project.
-
-- Locate the email identifier of the access key and pick the vertical dot menu at the far right of the row.
-- Select "_Create Key_" from the pop-up menu.
-- Select "JSON" from the Dialog Box.
-
-A proper key file will be downloaded to your local directory. It's important to safeguard that key file. For this example, we're going to name the file
-`service-account.json`.
-
-The proper key file is in JSON format. An example file is provided below, with private information replaced by "`...`"
-
-```json
-{
-  "type": "service_account",
-  "project_id": "...",
-  "private_key_id": "...",
-  "private_key": "...",
-  "client_email": "...",
-  "client_id": "...",
-  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
-  "token_uri": "https://oauth2.googleapis.com/token",
-  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
-  "client_x509_cert_url": "..."
-}
+The correct way to authenticate with Spanner is by generating an OAuth token and pointing your local application server to the token. In order for this to work, your Google Cloud must have the correct permissions; contact the Ops team to ensure the correct permissions are added to your account.
+
+First, install the Google Cloud command-line interface by following the instructions for your operating system [here](https://cloud.google.com/sdk/docs/install). Next, run the following to log in with your Google account (this should be the Google account associated with your Mozilla LDAP credentials):
+```sh
+gcloud auth application-default login
+```
+The above command will prompt you to visit a webpage in your browser to complete the login process. Once completed, ensure that a file called `application_default_credentials.json` has been created in the appropriate directory (on Linux, this directory is `$HOME/.config/gcloud/`). Now, create a symbolic link from this file to a file called `service-account.json` in the root of the repository:
+```sh
+ln -s /path/to/application_default_credentials.json /path/to/syncstorage-rs/service-account.json
```

Comment:
I don't think this symlink is needed? Google's auth seems to pick up mine from the `~/.config/` location.

It looks like the original docs referenced `service-account.json` in the root directory to match the setup in `Makefile`

Reply:
:face I'll update the README to reflect this. Do you think it's worth updating `Makefile` to remove `GOOGLE_APPLICATION_CREDENTIALS`? If there's anybody authenticating with a service account, their auth would break until they set up app default credentials

Reply:
It might be worth modifying the README to at least note the default path, but I think it's still good to include this bit too. Folk "in a hurry" tend not to read that deep, so answering questions now means not having to answer them later when they file issues.

Reply:
For some reason, the email notification for this thread went into my spam folder so I'm just seeing it now 🤦🏻 Just to make sure I understand, you're saying it might make sense to include info about authenticating via application default creds in addition to authenticating with a service account?

Reply:
I know this is a month old, but has this been addressed? Just wondering if we can clear the backlog a bit.

Reply:
Sorry for the delay! I updated the docs to include information about authenticating via both OAuth and a service account